README.md

```diff
@@ -15,60 +15,123 @@
 [![Follow Us](https://img.shields.io/twitter/url/https/secureteatool.svg?label=SecureTea%20Project&style=social)](https://twitter.com/secureteatool)
 [![All Contributors](https://img.shields.io/badge/all_contributors-7-orange.svg?style=flat-square)](#contributors)
 
-**The OWASP SecureTea Project** is an application designed to help secure a person's laptop or computer / server with IoT (Internet Of Things) and notify users (via various communication mechanisms), whenever someone accesses their computer / server. This application uses the touchpad/mouse/wireless mouse to determine activity and is developed in Python and tested on various machines (Linux, Mac & Windows).<br>
-The software is still under development, and will eventually have it's own IDS(Intrusion Detection System) / IPS(Instrusion Prevention System), firewall, anti-virus, intelligent log monitoring capabilities with web defacement detection, and support for much more communication medium.
+**The OWASP SecureTea Project** provides a one-stop security solution for various devices (personal computers / servers / IoT devices).<br>
+The **core** feature includes the following:
+- [x] Intrusion Detection System
+- [x] Firewall
+- [x] AntiVirus
+- [x] Server Log Monitor
+- [x] System Log Monitor
+- [x] Local Web Deface Detection & Prevention System
+- [x] Auto Web Server Patcher
+- [x] IoT Anonymity checker
+- [x] Auto report generation using OSINT
+- [x] Notifying suspicious activities using various mediums (Twitter, Telegram, Slack, Gmail, SMS, AWS & more)
+- [x] Interactive GUI for ease of setting up
 
 ![Setup Intro][Setup-help-animation]
 
 ## Table Of Contents
--   [Target User](#target-user)
--   [Objective](#objective)
 -   [Pre-requisites](#pre-requisites)
 -   [Installation Procedure](#installation-procedure)
+-   [Quick Start](#quick-start)
 -   [Suggestions and Contribution](#suggestions-and-contributing)
 -   [Code Of Conduct](/CODE_OF_CONDUCT.md)
--   [Chat Group](#chat-group)
 -   [User guide](/doc/en-US/user_guide.md)
 -   [Developer guide](/doc/en-US/dev_guide.md)
 -   [Translation-Japanese](/doc/ja-JP/README.md)
 -   [Translation-Hindi](/doc/hi-IN/README.md)
+-   [License](#license)
+
+## Pre-requisites
+#### Supported Platforms
+OWASP SecureTea Tool project runs on Linux operating systems.
+
+#### Hardware
+-  Linux OS / Raspberry Pi - have `sudo` access on the terminal/console
+-  Mouse / Wireless Mouse / Touchpad congenital laptop
+
+#### Software
+-  Python 2.x or 3.x
+-  Angular
+-  Twitter account (optional)
+-  Telegram account (optional)
+-  Slack account (optional)
+-  Twilio SMS account (optional)
+-  Amazon Web Services account (optional)
+-  Libnetfilter
+-  Yara
+-  Clam AV
+
+#### Installing pre-requisites
+Python:<br>
+https://www.python.org/
```

Comment:
Please make it hyperlink. Don't use the links as such. 

`[Python](https://www.python.org/)`